integration_tests/integration_tests.feature

```diff
@@ -50,6 +50,16 @@ Feature: Run integration tests
       |      two_sided_market.default_2a       |
       |    two_sided_pay_as_clear.default_2a   |
 
+  Scenario Outline: Test Cumulative Bills
+    Given we have a scenario named <scenario>
+    And d3a is installed
+    When we run the simulation with setup file <scenario> and parameters [24, 60, 60, 1]
+    Then the cumulative energy bills for each area are the sum of its children
+  Examples: Settings
+      |                scenario                |
+      |               default_2a               |
```

Comment:
As I intended in the D3ASIM-2303 we should include a setup file that has non zero grid fees. Thanks!

Reply:
Actually, after watching it, I did not include the grid fees in it (was not required). They were included implicitly though through the spent_total, therefore adding these here.